client/.storybook/preview.js

```diff
@@ -1,3 +1,15 @@
+import React from "react";
+import GlobalStyles from "../src/GlobalStyles";
 export const parameters = {
   actions: { argTypesRegex: "^on[A-Z].*" },
 };
+const withGlobalStyles = (Story, context) => {
+  return (
+    <>
+      <GlobalStyles />
+      <Story {...context} />{" "}
```

Comment:
Prettier artifact (happens sometimes on format) 
```suggestion
      <Story {...context} />
```